corehq/apps/app_manager/templates/app_manager/v2/apps_base.html

```diff
@@ -60,6 +60,23 @@
 </div>
 <div class="js-unhide-on-unsaved appmanager-content appmanager-content-transition-all{% if not formdesigner %} appmanager-settings-content {% endif %}"
      id="js-appmanager-body">
+    {% if request|ui_notify_enabled:"APP_BUILDER_RELEASE" and not formdesigner %}
+    <div class="alert alert-ui-notify alert-dismissible helpbubble helpbubble-purple helpbubble-no-arrow fade in"
+         data-slug="{{ "APP_BUILDER_RELEASE"|ui_notify_slug }}"
+         role="alert">
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+        </button>
+        <p class="lead">{% trans 'New App Builder Officially Released' %}</p>
+        <p>
+            {% blocktrans %}
+                After months of testing and feedback, we are happy to announce the
+                official release of the new CommCare App Builder. The previous App Builder
+                interface is now no longer available.
```

Comment:
This would be a nice place to link to the blog for people who've been ignoring v2 and don't know what it is: https://www.dimagi.com/blog/commcare-feature-update-new-app-manager/